scripts/external_data/load_baseball_savant.py

```
@@ -0,0 +1,306 @@
+        return psycopg2.connect(
+            host=parsed.hostname,
+            port=parsed.port or 5432,
+            database=parsed.path.lstrip('/'),
+            user=parsed.username,
+            password=parsed.password
+        )
+    
+    # Fall back to PG* variables
+    return psycopg2.connect(
+        host=os.getenv('PGHOST', 'localhost'),
+        port=os.getenv('PGPORT', 5432),
```

Ruff (0.15.10)

[warning] Invalid type for environment variable default; expected `str` or `None`

(PLW1508)